src/components/Application/rainbowLegacyStyles.js

```diff
@@ -1014,3 +1018,16 @@ export default `
   .rainbow-font-size-text_large {
     font-size: 1rem; }  
 `;
+
+const GlobalStyles = createGlobalStyle`
+    ${legacyStyles}
+`;
```

Comment:
let's put this inside the ```if (isServer) {``` block thus it is only created when is in server

Reply:
i did this 

```js
const RainbowLegacyStyles = isServer
    ? createGlobalStyle`
    ${legacyStyles}
`
    : () => <style>{legacyStyles}</style>;

export default RainbowLegacyStyles;
```
cos i a not sure how will be the impact on performance if we put `createGlobalStyle` insinde component function